src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.runtimeconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "runtimeOptions": {
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "1.0.0-rc3-002733"
```

Comment:
it is required as it is the way to tell the CLI that the app must run using the shared framework as opposed to just trying to load the dependencies only from the current folder. However, it looks like if your runtimeconfig references a lower version compared to the one that comes with the cli you are using to run, it looks like this should work as well. Seems like the only scenario where it would break, is if the version of the framework in your runtimeconfig is higher than the shared framework that comes with the cli that you are trying to run your app with, which I guess it kind of makes sense.